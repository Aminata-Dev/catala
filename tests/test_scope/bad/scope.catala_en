## Article

```catala
declaration scope A:
  context a content integer
  context b content integer
  context c content boolean

scope A:
  definition c equals false
  definition a under condition c consequence equals 42
  definition a under condition not c consequence equals 0
  definition b under condition not c consequence equals 1337
  definition b under condition not c consequence equals 0
```

```catala-test-inline
$ catala Interpret -s A
[WARNING] In scope "A", the variable "a" is never used anywhere; maybe it's unnecessary?

┌─⯈ tests/test_scope/bad/scope.catala_en:5.10-11:
└─┐
5 │   context a content integer
  │           ‾
  └─ Article
[WARNING] In scope "A", the variable "b" is never used anywhere; maybe it's unnecessary?

┌─⯈ tests/test_scope/bad/scope.catala_en:6.10-11:
└─┐
6 │   context b content integer
  │           ‾
  └─ Article
[ERROR] There is a conflict between multiple valid consequences for assigning the same variable.

This consequence has a valid justification:
┌─⯈ tests/test_scope/bad/scope.catala_en:13.56-60:
└──┐
13 │   definition b under condition not c consequence equals 1337
   │                                                         ‾‾‾‾
   └─ Article

This consequence has a valid justification:
┌─⯈ tests/test_scope/bad/scope.catala_en:14.56-57:
└──┐
14 │   definition b under condition not c consequence equals 0
   │                                                         ‾
   └─ Article
#return code 255#
```
