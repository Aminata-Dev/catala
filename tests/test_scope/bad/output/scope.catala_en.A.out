[ERROR] There is a conflict between multiple exceptions for assigning the same variable.

This justification is true:
  --> test_scope/bad/scope.catala_en
   | 
14 |   definition b under condition not c consequence equals 0
   |                                ^^^^^
   + Article

This justification is true:
  --> test_scope/bad/scope.catala_en
   | 
13 |   definition b under condition not c consequence equals 1337
   |                                ^^^^^
   + Article
